packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -88,21 +78,80 @@ export class CfnInclude extends core.CfnElement {
     return ret;
   }
 
+  /**
+   * Returns the CfnParameter object from the 'Parameters'
+   * section of the included template
+   * Any modifications performed on that object will be reflected in the resulting CDK template.
+   *
+   * If a Parameter with the given name is not present in the template,
+   * throws an exception.
+   *
+   * @param parameterName the name of the parameter to retrieve
+   */
+
+  public getParameter(parameterName: string): core.CfnParameter {
+    const ret = this.parameters[parameterName];
+    if (!ret) {
+      throw new Error(`Parameter with name '${parameterName}' was not found in the template`);
+    }
+    return ret;
+  }
+
+  /**
+   * Returns the CfnParameter object from the 'Parameters'
+   * section of the CloudFormation template with the given name.
+   * Any modifications performed on that object will be reflected in the resulting CDK template.
+   *
+   * If a Parameter with the given name is not present in the template,
+   * throws an exception.
+   *
+   * @param logicalId the name of the Parameter in the CloudFormation template file
+   */
+  /* public getParameter(parameterName: string): core.CfnParameter {
+    const ret = this.parameters[parameterName];
+    if (!ret) {
+      throw new Error(`Parameter with name '${parameterName}' was not found in the template`);
+    }
+    return ret;
+  }
+  */
+
   /** @internal */
   public _toCloudFormation(): object {
     const ret: { [section: string]: any } = {};
 
     for (const section of Object.keys(this.template)) {
       // render all sections of the template unchanged,
-      // except Conditions and Resources, which will be taken care of by the created L1s
-      if (section !== 'Conditions' && section !== 'Resources') {
+      // except Conditions, Resources, and Parameters, which will be taken care of by the created L1s
+      // ToDo
+      if (section !== 'Conditions' && section !== 'Resources' && section !== 'Parameters') {
         ret[section] = this.template[section];
       }
     }
 
     return ret;
   }
 
+  private createParameter(logicalId: string): void {
+    const expression = cfn_parse.FromCloudFormation.parseValue(this.template.Parameters[logicalId]);
+    const expressionCaseCorrected = {
+      type: expression.Type,
+      default: expression.Default,
+      allowedPattern: expression.AllowedPattern,
+      constraintDescription: expression.ConstraintDescription,
+      description: expression.Description,
+      maxLength: expression.MaxLength,
+      maxValue: expression.MaxValue,
+      minLength: expression.MinLength,
+      minValue: expression.MinValue,
+      noEcho: expression.NoEcho,
+    };
+    const CfnParameter = new core.CfnParameter(this, logicalId, expressionCaseCorrected);
```

Comment:
Variables are always lowerCase in TypeScript/JavaScript, so this should be `cfnParameter`.